drivers/clock_control/clock_stm32l0_l1.c

```diff
@@ -38,7 +38,8 @@ void config_pll_init(LL_UTILS_PLLInitTypeDef *pllinit)
  */
 void config_enable_default_clocks(void)
 {
-#if defined(CONFIG_EXTI_STM32) || defined(CONFIG_USB_DC_STM32)
+#if defined(CONFIG_EXTI_STM32) || defined(CONFIG_USB_DC_STM32) || \
+	(CONFIG_SOC_SERIES_STM32L0X && ENTROPY_STM32_RNG)
```

Comment:
I don't think symbol `ENTROPY_STM32_RNG` worked (should be `CONFIG_ENTROPY_STM32_RNG`).
If your samples worked with this piece of code, then maybe it is not needed.

Reply:
Whoops. The clock is definitely needed, but it was enabled because `CONFIG_EXTI_STM32` was set.

Reply:
Indeed.
Though it still needs to be fixed!